packages/deployer/ens.js

```diff
@@ -34,16 +35,79 @@ class ENS {
     this.ens.registryAddress = ensRegistry.address;
     this.devRegistry = ensRegistry;
     this.setENSJS();
+    await this.deployNewDevReverseRegistrar(from);
     return ensRegistry;
   }
 
+  //this method should only be called when using a dev registry!
+  //do not call it otherwise!
+  async deployNewDevReverseRegistrar(from) {
+    const registryAddress = this.determineENSRegistryAddress();
+    debug("from: %s", from);
+    debug("registryAddress: %s", registryAddress);
+    const ReverseRegistrarArtifact = require("./builtContracts/ReverseRegistrar");
+    const ReverseRegistrar = contract(ReverseRegistrarArtifact);
+    ReverseRegistrar.setProvider(this.provider);
+    //note: the resolver address we're supposed to pass in to the constructor
+    //is supposed to be the "default resolver"; I'm not sure what that means,
+    //but I figure using the resolver for "addr.reverse" ought to suffice,
+    //right?  So let's set up a resolver for it.
+    const reverseResolutionSpecialName = "addr.reverse";
+    debug("made it here");
+    //but in order to set its resolver, we first have to set its owner.
+    await this.setNameOwner({ from, name: reverseResolutionSpecialName });
+    debug("owner set");
+    //now we can actually set the resolver
+    const { resolverAddress } = await this.ensureResolverExists({
+      from,
+      name: reverseResolutionSpecialName
+    });
+    debug("resolver set: %s", resolverAddress);
+    //...but wait!  we need it to be owned by the registry, not by us.
+    //(otherwise the deployment will revert.)  so, let's hand over
+    //ownership to the registry.
+    await this.alienateNameOwner({
+      from,
+      owner: registryAddress,
+      name: reverseResolutionSpecialName
+    });
+    debug("owner alienated");
+    //now we can do the deployment!
+    const reverseRegistrar = await ReverseRegistrar.new(
+      registryAddress,
+      resolverAddress,
+      {
+        from
+      }
+    );
+    debug("deployed");
+    //except, we're not done... we need to transfer ownership from the registry
+    //to the reverse registrar.
+    //(if there were a previous reverse registrar, this would happen automatically,
+    //but there wasn't, so it doesn't.)
+    //so, first let's claim the name for ourself again
+    await this.setNameOwner({ from, name: reverseResolutionSpecialName });
```

Comment:
This is another way that I'm worried I overcomplicated things. :)  I think if we didn't give away `"reverse"`, like you suggest above, this wouldn't be necessary.  Since we did give it away, that leads to this mess where we have to get ti back rather than directly change things...

I guess I'll go change this to get rid of all of this unnecessary fiddling about.